features/step_definitions/authorization.js

```diff
@@ -34,6 +34,10 @@ defineSupportCode(({ Before, Given, Then }) => {
             case 'calvin':
                 Assert.strictEqual(decodedToken.username, this.username);
                 break;
+            case 'github:calvin':
+                Assert.strictEqual(decodedToken.username, this.username);
+                Assert.strictEqual(decodedToken.scmContext, this.scmContext);
```

Comment:
> So we will add the scmContext to be part of the jwt token profile later?

Yes, if we support multiple SCMs feature, we can't determine the user only by username, so I think jwt token profile should also contain `scmContext`.

> And how can we tell what scmContext the user is on when they first log in?

I think we can tell `scmContext` by API path.
Currently, the path of the login API is `/auth/login/{web?}`.
https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/auth/login.js#L15

For multiple SCMs, we have to change the API path like as `/auth/login/{scmContext?}/{web?}`.
`{scmContext}` parameter specifies the scm by which user want to log-in.